### PR TITLE
Changes to the CSS page. links, wordsmithing and html code highlighting

### PR DESCRIPTION
**Pull request by garster**

### CODE REVIEW

**Comment by garster** [comment]
@loganfranken I think I got it all.

**Comment by loganfranken** [comment]
Hey @garster! Thanks for your work on this!

The only thing is it looks like some of the bad practice examples got corrected to be good practice examples?

**Comment by garster** [comment]
@loganfranken do you mean when I changed the `.red` color from #CC0000 to #C00? 

I changed it because both the good and bad examples had the long-form. The example was about the name of the class. I feel the rest of the content which is only there to fill out the example, good or bad, should be good form. Many times I have copied some code like this and got bad stuff.

**Comment by loganfranken** [comment]
Ahh gotcha, my bad: I see you corrected it on the bad practice example that was focused on a different concept. Nevermind! Thanks for humoring me!

File: css.md [hunk 1]
```diff
@@ -12,19 +12,18 @@ More specifically, *websites should not convey information by color alone*.
...
-Also note that there may be inconsistencies between how the various browsers
-interpret CSS and display your website to users. This could be a result of a
-lack of support for certain CSS properties as well as conflicting or
-inconsistent implementations of CSS properties. You will need to test in the
-various browsers to ensure that your website design is
-[cross-browser compatible](/browsers/).
+CSS techniques and technologies are evolving rapidly. Website developers should
+try to understand the tradeoffs involved with their choices (loading one vs. many
+stylesheets, performance vs. organization, basic CSS vs. minified/compressed/compiled
+CSS). Website developers may want to consider using stylesheet minifies, compilers,
```
Comment (dsaludares) [inline css.md]: Change "minifies" to "minifiers"

Comment (dsaludares) [inline css.md]: Change "minifies" to "minifiers"

**Review by dsaludares — CHANGES_REQUESTED** [review]
Found one typo

**Pull request merged**